test/gtest/uct/ib/test_ib.cc

```diff
@@ -261,21 +261,11 @@ class test_uct_ib_gid_idx : public test_uct_ib_with_specific_port {
             UCS_TEST_SKIP_R(device_str.str() + " is empty");
         }
 
-        struct ibv_ah_attr ah_attr;
-        memset(&ah_attr, 0, sizeof(ah_attr));
-        ah_attr.dlid           = m_port_attr.lid;
-        ah_attr.port_num       = m_port;
-        ah_attr.is_global      = 1;
-        ah_attr.grh.dgid       = gid;
-        ah_attr.grh.sgid_index = gid_index;
-        ah_attr.grh.hop_limit  = 255;
-
-        struct ibv_ah *ah = ibv_create_ah(ib_md->pd, &ah_attr);
-        if (ah == NULL) {
-            UCS_TEST_SKIP_R("failed to create address handle on " + device_str.str());
+        if (!uct_ib_device_test_roce_gid_index(&ib_md->dev, m_port, &gid,
```

Comment:
would not it fail on assert in `uct_ib_device_test_roce_gid_index`?

Reply:
the test checks port is roce: https://github.com/openucx/ucx/pull/4085/files/4bd1e44f12fee67fb4286036eb3e2caecb40492e#diff-f639bf578bf86fa86c73a758832ec754R230